### PR TITLE
Update form styles

### PR DESCRIPTION
**Pull request by dcmouyard**
I updated the default form styles to match the [U.S. Web Design Standards](https://standards.usa.gov/form-controls/).

This pull request depends on #46, #47, and #51 being merged first.

### CODE REVIEW

File: sass/partials/components/_form-item.scss [hunk 1]
```diff
@@ -45,22 +64,254 @@ $form-text-size: $font-size-md !default;
...
+  & + .form-item__label:before {
+    border-radius: rem(3px);
+    box-shadow: 0 0 0 1px #757575;
+    height: rem(18px);
+    line-height: rem(18px);
+    width: rem(18px);
+  }
+
+  &:checked + .form-item__label:before {
+    @include svg-background(correct);
```
Comment (coreylafferty) [inline sass/partials/components/_form-item.scss:173]: Does this need a background-size property?  height/width?

File: images/bg/arrow-down.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" viewBox="0 0 444.8 444.8"><path d="M248.1 352L434 165.9c7.2-6.9 10.8-15.4 10.8-25.7 0-10.3-3.6-18.8-10.8-25.7l-21.4-21.7c-7-7-15.6-10.6-25.7-10.6-9.9 0-18.6 3.5-26 10.6L222.4 231.5 83.7 92.8c-7-7-15.6-10.6-25.7-10.6-9.9 0-18.6 3.5-26 10.6l-21.4 21.7c-7 7-10.6 15.6-10.6 25.7s3.5 18.7 10.6 25.7L196.4 352c7.4 7 16.1 10.6 26 10.6 10.1 0 18.7-3.5 25.7-10.6z"/></svg>
```
Comment (coreylafferty) [inline images/bg/arrow-down.svg]: height/width here is pretty big (which makes the PNG fallback huge -- and anything using PNG fallback won't support background-size).

**Comment by coreylafferty** [comment]
See inline code comments.

**Comment by dcmouyard** [comment]
I changed the height/width of the select down arrow to 20px and set the background size to the checkbox check.

**Pull request merged**